src/components/Calculator.js

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
- [ ] It will be great to fix the linter error instead of disabling it. Remember the linter is supposed to help you write clean code.

Reply:
Hello @peterrobert, when I use function component, I can use as stateless function but in the requirements I need to implement with class component. class component is a stateful/container component. That is why It will cause this error.
I have two choices the first one is commented eslint and the second one is using function component.